src/command-line/outdated.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+const program = require("commander");
+const Utils = require("./utils");
+const packageManager = require("../packageManager");
```

Comment:
Error: Cannot find module '../packageManager'

Reply:
Yeah, fixed this.